### ISSUE TITLE
Reload config

### ISSUE THREAD

**Issue by Khailz** [flyth/ts3soundbot#75]
Instead of restarting to reload a new config file change I think it would be good to be able to just reload from interface or chat to test settings. 

But I have a feeling this would be hard to implement with so much progress done already.

**Comment by MrJoki007** [comment]
I think its not necessary, because you may edit the config of the bot at the setup and you dont need to edit it anymore

**Comment by Khailz** [comment]
There are different levels and true or false values that you would need to restart to see changes for, and with constant updates then restarting all the time would be a bit tedious.

**Comment by flyth** [comment]
A full reload won't be possible since some settings are required for the bot to startup correctly (like paths & such). Other settings might be overridden from the interface, like debug level & such.
Could you name the specific settings you would want to change at runtime?

**Comment by Khailz** [comment]
Yeah I knew a full wouldn't be possible but yes things such as level, and mostly any true and false value. The only reason I say this is because there is no auto start that starts off with a playlist.
